src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -402,6 +405,50 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
+        internal async Task HandleSaveSkuRecommendationsRequest(
+            SaveSkuRecommendationsParams parameters,
+            RequestContext<SaveSkuRecommendationsResult> requestContext)
+        {
+            const string SkuRecommendationJsonFileExt = ".json";
+            const string SkuRecommendationHtmlFileExt = ".html";
+            string timeStamp = DateTime.UtcNow.ToString("yyyyMMddHHmmss");
```

Comment:
is `CultureInfo.InvariantCulture` needed here?

Reply:
always good to have.